docs/configuration/settings.rst

```diff
@@ -5,113 +5,469 @@ Mautic Configuration Settings
+
+
 CORS settings
 =============
 
+Cross-Origin Resource Sharing (CORS) enables data to pass between your website and Mautic.
+
+.. image:: images/cors-settings.png
+  :width: 600
+  :alt: Screenshot showing CORS Settings Configuration in Mautic
+
+* **Restrict Domains** - When set to No, any web page can pass information to Mautic. Select Yes to limit communication with your Mautic instance to websites listed in Valid Domains.
+
+* **Valid Domains** - A list of domains allowed to communicate with your Mautic instance. In the text box, list the exact URL of the top level domain you want to enable, one per line. For example: ``https://www.mautic.com`` tracks any activity on Mautic.com pages, but ``https://www.mautic.com`` won’t because Mautic.com is a secure website.
+
+.. note:: 
+
+  In the Valid Domains field, don’t include a slash at the end. For example, use ``https://www.mautic.com`` instead of ``https://www.mautic.com/``.
+
 Miscellaneous settings
 ======================
 
+.. image:: images/miscellaneous-settings.png
+  :width: 600
+  :alt: Screenshot showing Miscellaneous Settings Configuration in Mautic
+
+* **Trusted hosts** - To explicitly whitelist the hosts allowed to send requests to Mautic. You can use regular expression and separate multiple hosts with a comma. i.e ``.*\.?trusted.com$`` If left empty, Mautic will respond to all hosts.
+
+* **Trusted proxies** - To configure the IP addresses that should be trusted as proxies. This setting is mandatory when using Mautic behind an SSL terminating proxy. Separate multiple IP addresses by a comma. i.e ``127.0.0.1, 10.0.0.0/8, fc00::/7``
+
+* **IP lookup service** - By default, Mautic uses MaxMind GeoLite2 City to identify the city of a website visitor, based on the location of the Internet Service Provider (ISP) for their IP address.
+
+* **IP lookup service authentication** - To use another available IP lookup service, enter your credentials.
+
+* **List of IPs not to track Contacts with** - To turn-off tracking for particular IP addresses, enter the addresses, one per line. Mautic doesn’t recommend adding your office IP address. If you list your internal IP address, Mautic won’t track clicks, page hits, etc., from that IP, including testing clicks.
+
+* **List of Bots to not track with** - Mautic has identified and turn-off tracking for several known bots. To track activity from those bots, remove them from this list. To turn-off tracking for other bots, add them here (one per line).
+
+* **URL Shortener** - If you use a URL shortening service like bit.ly for links in SMS messages, enter your access token here. For bit.ly, use:
+
+.. note:: 
+
+    https://api-ss..bitly.com/v3/shorten?access_token=[ACCESS_TOKEN]&format=txt&longUrl
```

Comment:
Please enter the full URL - this has been shortened which does not help the user.